sematic/ui/packages/common/src/pages/RunDetails/metrics/MetricsPane.tsx

```diff
@@ -0,0 +1,102 @@
+import { Timeline } from "@mui/icons-material";
```

Comment:
Will there be another PR for pipeline metrics?

Reply:
Yes.